beetsplug/gmusic.py

```diff
@@ -59,7 +58,9 @@ def upload(self, lib, opts, args):
         items = lib.items(ui.decargs(args))
         files = [x.path.decode('utf-8') for x in items]
         ui.print_(u'Uploading your files...')
-        self.m.upload(filepaths=files)
+        pool = multiprocessing.Pool(initializer=self._pool_init)
+        chunks = [tuple(files[i:i + 25]) for i in range(0, len(files), 25)]
```

Comment:
It would be nice to have the chunk size configurable by the user in their `config.yaml` file.

Reply:
We could, but there isn't much value in changing it to be honest.

 Configuring the number of processes in the pool night be worth it though.

Reply:
Yeah, that sounds like it would be useful.